### PR TITLE
fix(types): make webhooks optional

### PR DESCRIPTION
**Pull request by StarpTech**
#### Checklist

- [ ] run `make test`
- [ ] PR title must follow [conventional-commit-standard](https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md#conventional-commit-standard)
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Code of conduct](https://github.com/wundergraph/wundergraph/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by jensneuse** [comment]
should the title be "make webhooks tests optional"?

**Comment by StarpTech** [comment]
> should the title be "make webhooks tests optional"?

No, this PR makes webhooks in ConfigureWunderGraph types optional. I could not run the tests locally because integration tests are run by default. That behavior is also addressed in this PR.

**Review by jensneuse — APPROVED** [review]

**Review by jensneuse — APPROVED** [review]

File: pkg/s3uploadclient/s3uploadclient_test.go [hunk 1]
```diff
@@ -10,6 +11,10 @@ import (
 )
 
 func TestS3UploadClient_UploadFile(t *testing.T) {
+	if os.Getenv("INT") != "true" {
```
Comment (spetrunin) [inline pkg/s3uploadclient/s3uploadclient_test.go:14]: does it make sense to use [short](https://pkg.go.dev/testing#hdr-Skipping)?



example:
https://github.com/wundergraph/graphql-go-tools/pull/382/commits/2b606d9bfb33992ad28c54f57b6ce6df1861d7d2

Comment (StarpTech) [reply pkg/s3uploadclient/s3uploadclient_test.go:14]: Good point. Would you like to create a PR? Otherwise, I'll tackle this next time.

**Pull request merged**